.github/ISSUE_TEMPLATE/deployment.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Deployment
+about: Use this template when you're going to deploy a new release of the extension and web app.
+title: Publish vX.X.X extension + web app
+labels: deployment
+---
+
+- [ ] Manual testing to confirm extension works with main flows
+  - [ ] Balances
+  - [ ] Send
+  - [ ] Swap
+  - [ ] Staking
+  - [ ] IBC
+  - [ ] Tx details
+  - [x] ~Do a full sync with with [assertRootValid()](https://github.com/penumbra-zone/web/blob/main/packages/query/src/block-processor.ts#L383-L395) enabled. Will throw if TCT roots are not correct.~ (Ignore for now.)
+- [ ] If there is already a changelist PR (like [this one](https://github.com/penumbra-zone/web/pull/799)), merge it. Otherwise, manually run `pnpm changeset version`.
```

Comment:
@grod220 does something need to be committed after running `pnpm changeset version`? If so, does that require a separate PR that must be reviewed and merged before continuing the deployment?

Reply:
Cool, done